src/Response/Context.php

```diff
@@ -65,6 +65,15 @@ public function isFailure(): bool
         return $statusCode !== min(max($statusCode, 200), 208); // [200,208] = HTTP OK
     }
 
+    /**
+     * Is response body missing.
+     */
+    public function isBodyMissing(): bool
+    {
+        $rawBody = $this->response->getRawBody();
+        return $rawBody === '';
```

Comment:
Blocker: As we agreed upon checking the whitespaces in the body as well therefore the rawBody should be trimmed here.

Reply:
Added changes